Bottom: 14b9068e4a5ec183da036f36a8b8c54bb762f809
Top:    30a620cfa32a0bc20c11353f155747707b0251f6
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-28 23:48:02 -0400

Unbreak the mpc5200 i2s Kconfig

Unbreak the mpc5200 i2s Kconfig

Signed-off-by: Jon Smirl <jonsmirl@gmail.com>


---

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index c7339f3..dc79bdf 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -24,7 +24,7 @@ config SND_SOC_MPC8610_HPCD
 
 config SND_SOC_MPC5200_I2S
 	tristate "Freescale MPC5200 PSC in I2S mode driver"
-	depends on PPC_MPC52xx && PPC_BESTCOMM && BROKEN
+	depends on PPC_MPC52xx && PPC_BESTCOMM
 	select SND_SOC_OF_SIMPLE
 	select SND_MPC52xx_DMA
 	select PPC_BESTCOMM_GEN_BD
